sonar-dotnet-shared-library/src/test/java/org/sonarsource/dotnet/shared/plugins/protobuf/RazorSymbolRefsImporterTest.java

```diff
@@ -19,61 +19,60 @@
  */
 package org.sonarsource.dotnet.shared.plugins.protobuf;
 
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.nio.file.Paths;
 import org.junit.Before;
-import org.junit.Rule;
 import org.junit.Test;
 import org.slf4j.event.Level;
-import org.sonar.api.batch.fs.internal.FileMetadata;
-import org.sonar.api.batch.fs.internal.TestInputFileBuilder;
-import org.sonar.api.batch.sensor.internal.SensorContextTester;
-import org.sonar.api.notifications.AnalysisWarnings;
-import org.sonar.api.testfixtures.log.LogTester;
-
-import java.io.File;
-import java.io.FileNotFoundException;
-import java.io.FileReader;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Mockito.mock;
 import static org.sonarsource.dotnet.shared.plugins.ProtobufDataImporter.SYMBOLREFS_FILENAME;
 
-public class RoslynSymbolRefsImporterTest {
-
-  private static final File TEST_DATA_DIR = new File("src/test/resources/RazorProtobufImporter");
-  private static final String TEST_FILE_PATH = "Cases.razor";
-  private static final File TEST_FILE = new File(TEST_DATA_DIR, TEST_FILE_PATH);
-
-  private final SensorContextTester sensorContext = SensorContextTester.create(TEST_DATA_DIR);
+public class RazorSymbolRefsImporterTest extends RazorImporterTestBase {
   private final File protobuf = new File(TEST_DATA_DIR, SYMBOLREFS_FILENAME);
 
-  @Rule
-  public LogTester logTester = new LogTester();
-
+  @Override
   @Before
-  public void setUp() {
-    logTester.setLevel(Level.TRACE);
+  public void setUp() throws FileNotFoundException {
+    super.setUp();
     assertThat(protobuf).withFailMessage("no such file: " + protobuf).isFile();
   }
 
   @Test
-  public void test_symbol_refs_get_imported() throws FileNotFoundException {
-    var inputFile = new TestInputFileBuilder("dummyKey", TEST_FILE_PATH)
-      .setMetadata(new FileMetadata(mock(AnalysisWarnings.class)).readMetadata(new FileReader(TEST_FILE)))
-      .build();
-    sensorContext.fileSystem().add(inputFile);
+  public void test_symbol_refs_get_imported_cases() {
 
```

Comment:
pedantic: there is an empty line at the beginning of the tests that is not needed